src/api/app/views/webui/user/_info.html.haml

```diff
@@ -38,18 +38,9 @@
                                                                  'features we develop and give us feedback on them before they go live.' } } |
       .mt-4
         - if configuration.accounts_editable?(user)
-          - if account_edit_link.present?
-            = link_to(account_edit_link, class: 'd-block') do
-              %i.fas.fa-user-edit
-              Edit your account
-          - else
-            = link_to('#', data: { toggle: 'modal', target: '#edit-modal' }, class: 'd-block') do
-              %i.fas.fa-user-edit
-              Edit your account
+          = render partial: 'webui/user/edit_account', locals: { account_edit_link: account_edit_link }
         - if configuration.passwords_changable?(user)
-          = link_to('#', data: { toggle: 'modal', target: '#password-modal' }, class: 'd-block') do
-            %i.fas.fa-key
-            Change your password
+          = render partial: 'webui/user/change_password'
         = link_to(my_subscriptions_path, class: 'd-block') do
           %i.fas.fa-bell
           Change your notifications
```

Comment:
What is the reason for not having this link in the `Actions` if `Change your password` is there?

Reply:
Well, to me this is more like navigation... It sends you to the notifications page.

Reply:
I'm gonna change that also, because the label (`Change your notifications`) is asking for an action, clearly...